src/components/Person/index.jsx

```diff
@@ -0,0 +1 @@
+export * from './Person';
```

Comment:
There are no issues found in this file. The export statement correctly exports the 'Person' component.